### PR TITLE
Appium/add scenario changepassword

### PR DESCRIPTION
**Pull request by chrisleewilcox**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
Added change password scenario.

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Review by Andepande — APPROVED** [review]

**Review by cortisiko — CHANGES_REQUESTED** [review]
PR looks good overall, just left a few comments. I did notice the test was failing for me on Browserstack. I ran it against the pixel 6 running android 12.

File: wdio/screen-objects/ChangePasswordScreens.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return Selectors.getElementByPlatform(
+      RESET_PASSWORD_ANDROID_TERM_CHECKBOX_ID,
+    );
+  }
+  async typePassword(text) {
+    const elem = await this.passwordInput;
+    await elem.waitForDisplayed();
+    await Gestures.typeText(elem, text);
+  }
+  async tapCONFIRM() {
```
[the remark this answers is not in the document]
Comment (chrisleewilcox) [reply wdio/screen-objects/ChangePasswordScreens.js:26]: Updated.

File: wdio/screen-objects/ChangePasswordScreens.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  RESET_PASSWORD_INPUT_ID,
+  RESET_PASSWORD_CONFIRM_BUTTON_ID,
+  RESET_PASSWORD_ANDROID_TERM_CHECKBOX_ID,
+} from './testIDs/Screens/ChangePasswordScreensIDs.testIds';
+
+class ChangePasswordScreens {
+  get passwordInput() {
+    return Selectors.getElementByPlatform(RESET_PASSWORD_INPUT_ID);
+  }
+  get buttonCONFIRM() {
```
[the remark this answers is not in the document]
Comment (chrisleewilcox) [reply wdio/screen-objects/ChangePasswordScreens.js:13]: Updated.

File: app/components/Views/ResetPassword/index.js [hunk 3]
```diff
@@ -722,7 +726,11 @@ class ResetPassword extends PureComponent {
                     secureTextEntry={secureTextEntry}
                     placeholder={''}
                     placeholderTextColor={colors.text.muted}
-                    testID={CONFIRM_PASSWORD_INPUT_BOX_ID}
+                    // testID={CONFIRM_PASSWORD_INPUT_BOX_ID}
```
[the remark this answers is not in the document]
Comment (chrisleewilcox) [reply app/components/Views/ResetPassword/index.js:729]: Updated.

**Comment by chrisleewilcox** [comment]
Test result on browserstack:
https://app-automate.browserstack.com/dashboard/v2/builds/af352c3cb4fc61986127b52d3b21c37a2213b608/sessions/60b691472713286d50be20b8f33c7fe60e3ebb5c

**Comment by chrisleewilcox** [comment]
Browserstack ./features/LockResetWallet.feature
https://app-automate.browserstack.com/dashboard/v2/builds/af352c3cb4fc61986127b52d3b21c37a2213b608/sessions/6ceedb26595d573523adf353e9ef95220260402f

**Comment by chrisleewilcox** [comment]
Browserstack ./features/ChangePassword.feature
https://app-automate.browserstack.com/dashboard/v2/builds/af352c3cb4fc61986127b52d3b21c37a2213b608/sessions/2fece468aff3efe1283de3c52e2ed6cef8c01e20

**Comment by chrisleewilcox** [comment]
detox
https://app.bitrise.io/build/ec89e75c-b9b4-4ade-9a79-0efc1f6461f9

**Pull request closed without merging**